frontend/src/app/components/settings/settings.component.html

```diff
@@ -0,0 +1,40 @@
+<div style="text-align: center; display: flex; align-items: center; justify-content: center">
+	<mat-card>
+		<mat-card-header>
+			<mat-card-title>Settings</mat-card-title>
+		</mat-card-header>
+		<form [formGroup]="updateUserForm" (submit)="updateUserInfo()">
+			<mat-form-field appearance="fill">
+			  <mat-label>Username</mat-label>
+			  <input matInput formControlName="username">
+			</mat-form-field>
+			<mat-form-field appearance="fill">
+			  <mat-label>Full Name</mat-label>
+			  <input matInput formControlName="fullName">
```

Comment:
in addition to allowing users to change their full name, we should also allow them to change their "friendly name"